src/packages/migrate/src/commands/DbPull.ts

```diff
@@ -354,6 +343,18 @@ continue with the instructions below.`
       const modelsCount = (introspectionSchema.match(/^model\s+/gm) || [])
         .length
 
+      const prisma1UpgradeMessageBox = prisma1UpgradeMessage
+        ? '\n\n' +
+          drawBox({
+            height: 13,
+            width: 74,
+            str:
+              prisma1UpgradeMessage +
+              '\nOnce you upgraded your database schema to Prisma 2.0, \ncontinue with the instructions below.',
```

Comment:
Empty line at end for beauty:
```suggestion
              '\nOnce you upgraded your database schema to Prisma 2.0, \ncontinue with the instructions below.\n',
```